### PR TITLE
create admin dashboard part2

### PR DESCRIPTION
**Pull request by tung3457**

### CODE REVIEW

**Comment by tung3457** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -19,3 +19,19 @@
...
+Route::group(array('prefix' => 'admin', 'namespace' => 'Admin', 'middleware' => 'manager'), function () {
+    Route::get('/', 'PageController@home');
+    Route::get('users', 'UserController@index')->name('admin.user.index');
+    Route::get('roles', 'RoleController@index')->name('admin.role.index');
+    Route::get('roles/create', 'RoleController@create');
+    Route::post('roles/create', 'RoleController@store')->name('admin.role.create');
+    Route::delete('roles/{id?}/delete', 'RoleController@destroy')->name('admin.role.delete');
+    Route::get('users/{id?}/edit', 'UserController@edit')->name('admin.user.edit');
+    Route::post('users/{id?}/edit', 'UserController@update');
+    Route::delete('users/{id?}/delete', 'UserController@destroy')->name('admin.user.delete');
```
Comment (BunnyPi04) [inline routes/web.php]: sao ko dùng resource controller ấy?
https://stackoverflow.com/questions/25290229/laravel-named-route-for-resource-controller

**Comment by BunnyPi04** [comment]
commented

**Comment by tung3457** [comment]
ready

**Comment by tung3457** [comment]
ready

**Pull request merged**